Jenkinsfile

```diff
@@ -113,4 +104,4 @@ pipeline {
       }
     }
   }
-}
+}
```

Comment:
I think you should remove the other steps too!

Reply:
It breaks the integration test if I remove some of the other steps. If we can get that integration action merged in first I can give it another go.